doc/whats-new.rst

```diff
@@ -74,6 +74,8 @@ Bug fixes
   <https://github.com/josephnowak>`_.
 - Fix binning by multiple variables where some bins have no observations. (:issue:`9630`).
   By `Deepak Cherian <https://github.com/dcherian>`_.
+- Fix bug with `open_datatree(group='some_subgroup')` returning empty parent nodes. (:issue:`9665`, :pull:`9666`)
+  by `Alfonso Ladino <https://github.com/aladinor>`_.
```

Comment:
do we add what's new entries for datatree changes? I mean, we're pretty inconsistent, see the entry for disallowing slashes in variable names for `DataTree`, but I think we can just add @aladinor as a contributors to the main `DataTree` entry.